best-practices/README.md

```diff
@@ -101,6 +101,10 @@ Rails
   environment variables are detected on deploy.
 * [Use blocks][date-block] when declaring date and time attributes in FactoryGirl factories.
 * Use `touch: true` when declaring `belongs_to` relationships.
+* Name service classes after the action they perform
+(e.g. `CreatePurchase`, `UpdateCreditCard`).
+* Expose one public method on services that does the work
+(i.e. `run`, `call`, or `perform`).
```

Comment:
Is "i.e." intended to convey that it should be called one of these names, and not something else? Or are these only examples?

Could we agree on a single name for this?

How do we feel about `UpdateCreditCard.update()`-style naming?